### PR TITLE
Format

### PR DESCRIPTION
**Pull request by AurevoirXavier**
Apply the toml format.

### CODE REVIEW

File: core/inflation/Cargo.toml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-readme      = "README.md"
-authors     = { workspace = true }
-edition     = { workspace = true }
-version     = { workspace = true }
+authors.workspace = true
+description       = "Darwinia economic inflation mechanism implementation."
+edition.workspace = true
+name              = "dc-inflation"
+readme            = "README.md"
+version.workspace = true
```
Comment (boundless-forest) [inline core/inflation/Cargo.toml:7]: image

Reported by the [Better TOML](https://marketplace.visualstudio.com/items?itemName=bungcip.better-toml) 

II am not sure which approach is the best practice.

Comment (AurevoirXavier) [reply core/inflation/Cargo.toml:7]: That extension is sucks.

Comment (boundless-forest) [reply core/inflation/Cargo.toml:7]: Why?

Comment (AurevoirXavier) [reply core/inflation/Cargo.toml:7]:
```
Last updated | 2/18/2018, 4:50:44 PM
```

Comment (boundless-forest) [reply core/inflation/Cargo.toml:7]: Okay. I got it. Switch to another one.

**Review by boundless-forest — APPROVED** [review]

**Pull request merged**